models/validation-inference-scripts/root-cause-models/requirements.txt

```diff
@@ -0,0 +1,7 @@
+cudf==0+untagged.1.g788bddd
+cupy==9.1.0
+numpy==1.23.4
+onnxruntime==1.13.1
+scipy==1.6.0
+torch==1.7.1
```

Comment:
morpheus uses torch 1.10.2. should/can this be updated to same?

Reply:
updated the torch version.